vyper/context/types/function.py

```diff
@@ -279,19 +284,17 @@ def from_FunctionDef(
             raise StructureException("Cannot use reentrancy guard on view or pure functions", node)
 
         # call arguments
-        arg_count: Union[Tuple[int, int], int] = len(node.args.args)
+        min_arg_count = len(node.args.args)
```

Comment:
Move this here, since we know this ahead of time now
```suggestion
        min_arg_count = len(node.args.args) - len(node.args.defaults)
```